packages/blockchain-wallet-v4-frontend/src/modals/OrderMyCard/CreateCardStep/index.tsx

```diff
@@ -37,7 +37,7 @@ const CreateCardStep = ({ handleClose, handleCreateCard }: Props) => (
       {/* disabled for MVP, will need this for V1 */}
       {/* <SeeCardDetailsBtn onClick={handleShowCardDetails} /> */}
 
-      <Image name='order-my-card-icon' />
+      <Image name='order-my-card-icon' style={{ marginBottom: '32px' }} />
```

Comment:
`styled-component`?

Reply:
I prefer to leave just the style here instead of adding more lines for a simple margin

Reply:
Sure, but what if I add you to add margin top of 12px?

Reply:
I could add it to the “SubTexWrapper” that is up above